.github/workflows/black.yml

```diff
@@ -0,0 +1,13 @@
+name: Black Lint Check
```

Comment:
Use black in addition to flake8 validate code format

Reply:
@Linus045 does it automatically format your code within the particular PR?

Reply:
It does run black using the pre-commit when you commit changes, which formats the code to match flake8 standard.
You can also run black manually via the console command "black src".
See development_setup.md for more details

Reply:
I see, thank you! I just found this one https://github.com/marketplace/actions/run-black-formatter#commit-changes-or-create-a-pull-request and was curious of how it is working